### DISCUSSION TITLE
Difficulty cashing out / closing down

### DISCUSSION THREAD

**Discussion by MarkMD0OMY** [raspiblitz/raspiblitz#4370] [General]
Hi All,
I installed Raspiblitz onto a Proxmox Debian VM to have a play with it about a month ago.
I only got as far as sending a small amount of bitcoin to it (which took 2 weeks to be confirmed!)
No channels opened as I did a load of reading but its a bit too much for me so I just want to get my coins back into my desktop wallet and forget the whole idea...
However when trying to "Send" my coins back to my wallet I'm being prompted for a "Fee".
Whatever I seem to put in here I get an error message, not enough funds or something similar each time.
Can anyone point me in the right direction please?

Thanks...

**Comment by rootzoll** [comment]
What interface UI do you use? Have you tried one of the other options: RTL, Thunderhub oder RaspiBlitzWebUI?

**Reply by MarkMD0OMY** [reply]
Thanks for the reply.
I just use the standard Web management interface.
Are those other options just alternatives?

**Reply by rootzoll** [reply]
Yes under apps you should find RTL or Thunderhub which should give you more detailed options on sending funds.

**Reply by MarkMD0OMY** [reply]
Ok thanks, the last time I checked it, the Apps page wasn't working, just a spinning yellow circle for ever.
I'll take another look when I'm free and see if I can install one of the alternatives.

**Reply by MarkMD0OMY** [reply]
Just thought I'd update here...
After sorting out the Apps page not working due to needing 8.8.8.8 as an additional DNS server? (when it already had my 2 Piholes and ISP router in the settings?!) I installed Ride The Lightning.
I was able to recover all my funds using the "Sweep All" option.

Thanks for the help, this issue can be closed now.
